app/config/locales/en.yml

```
@@ -162,6 +162,7 @@ en:
         your_login_credentials: Your login credentials for the payroll provider account
     error_invalid_token: The invitation link used is not valid. Double check the link and try again. If you continue experiencing issues, contact your caseworker.
     error_missing_token_html: "<strong>Your session has timed out due to inactivity.</strong> To continue where you left off, please click the link you received from your benefits agency in your email. If you encounter any issues, contact your benefits agency for assistance."
+    error_no_access: You do not have access to this resource. You have been redirected to the entry page.
```

I think something a bit less stern/more user-friendly would be good in case we accidentally show this to a user through some fault of our own:

```suggestion
    error_no_access: We weren't able to load payroll data for this account. Please click the link you received from your benefits agency to try again.
```